network-api/networkapi/buyersguide/templates/fragments/pni_nav_links.html

```diff
@@ -1,4 +1,9 @@
-{% load bg_nav_tags i18n %}
+{% load bg_nav_tags i18n wagtailroutablepage_tags %}
 
-{% url 'about-why-view' as about_why_url %}
+{% if page %}
```

Comment:
following https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/buyersguide/templates/about/about_nav.html#L4-L19

Reply:
@Pomax 🙄

Reply:
That code only works for objects that extend the `RoutablePageMixin`, like the `BuyerGuidePage` (which needs to do so because it defines all the "virtual routes" for the about pages). Products aren't routable pages, so the `routablepageurl` tag won't work for them.

Reply:
Asking Kalob about this, his suggestion is to add this to our models that use the pni_nav_links:

```
def get_context(self, request, *args, **kwargs):
    context = super().get_context(request, *args, **kwargs)
    # ...if there's already context code, that goes here...
    context['about_page'] = BuyersGuidePage.objects.first()
    return context 
```

that way we can change the template from this
```
{% routablepageurl page 'about-why-view' as about_why_url %}
```
into this:
```
{% routablepageurl about_page 'about-why-view' as about_why_url %}
```

Reply:
Updated! I've only added `context['about_page'] = BuyersGuidePage.objects.first()` to `ProductPage` and `BuyersGuidePage`. Those `about` virtual routes seem fine without having `about_page` as context.